Assets/WebGLTemplates/Thirdweb/lib/thirdweb.js

```diff
@@ -4,124 +4,107 @@ import { ThirdwebSDK } from "https://esm.sh/@thirdweb-dev/sdk?bundle";
 
 // big number transform
 const bigNumberReplacer = (key, value) => {
-  // if we find a BigNumber then make it into a string (since that is safe)
-  if (
-    ethers.BigNumber.isBigNumber(value) ||
-    (typeof value === "object" &&
-      value !== null &&
-      value.type === "BigNumber" &&
-      "hex" in value)
-  ) {
-    return ethers.BigNumber.from(value).toString();
-  }
-  return value;
+    // if we find a BigNumber then make it into a string (since that is safe)
+    if (
```

Comment:
are these just formatting changes? for JS files we use prettier, if you can, install it too so we don't override styles over each other.

Also what formatting tool do you use for C# code? haven't found a good one yet.

Reply:
Yep mb had whitespace changes hidden by default, I used prettier with 4 tab width and 120 line length, what settings do you use so I can set it up? For C# I use the default C# Microsoft VS Code extension, I think it uses either Razor or the default C# formatter.